frontend/src/modules/entitydetails/components/Metadata.css

```diff
@@ -49,3 +49,19 @@
     margin: 0;
     padding: 0;
 }
+
+.truncate {
+    position: relative;
+}
+
+.metadata button {
+    background: none;
+    border: none;
+    color: #7BC876;
+    font-style: italic;
+    font-weight: 300;
+    padding: 0;
+    position: absolute;
+    top: 2px;
+    right: 4px;
```

Comment:
The button should just follow the text instead of being positioned absolute at the end of the line. So you should remove the last three properties and set `padding: 0 0 0 2px;`.

I also think `font-weight: 300;` doesn't have any impact?

Reply:
With this no longer set to absolute there is also no need for the `truncate` class with position of relative, so I removed it.

Reply:
Good call!